Pipfile

```diff
@@ -24,17 +24,22 @@ boto3 = "*"
 pexpect = "*"
 pysftp = "*"
 setuptools = "*"
-cryptography = "==3.3.2"
+cryptography = "==3.4.8"
 sqlalchemy = "==1.3.23"
 flask-openid = "==1.3.0"
 flask-sqlalchemy = "==2.4.4"
 jinja2 = "==2.11.3"
 werkzeug = "==0.16.1"
 marshmallow = "==2.21.0"
 wtforms = "==2.3.3"
+markupsafe= "==2.0.1"
+
+# required in circleci context.
+numpy= "*"
+
 
 [packages]
 pytest = "==6.2.5"
 
 [requires]
-python_version = "3.7"
+python_version = "3"
```

Comment:
This sets python version to 3 so that we can use 3.9 locally but still be able to use other versions  in production.  I'm not sure if it matters either since we install production deps via "Requirements.txt in the airflow deployment."